torch/utils/collect_env.py

```diff
@@ -260,17 +261,18 @@ def get_env_info():
 
     if TORCH_AVAILABLE:
         version_str = torch.__version__
-        debug_mode_str = torch.version.debug
-        cuda_available_str = torch.cuda.is_available()
+        debug_mode = torch.version.debug
+        cuda_available = torch.cuda.is_available()
         cuda_version_str = torch.version.cuda
     else:
-        version_str = debug_mode_str = cuda_available_str = cuda_version_str = 'N/A'
```

Comment:
Hmm, I think it's better to return that `Is CUDA Available` and `Is debug build` are `N/A` if PyTorch installation could not be found

Reply:
Okay, I'll have to change the values above from `bool` to `str` then. Will update.